### ISSUE TITLE
[PHP_PHPSTAN] default rule level should follow default PHPStan behavior

### ISSUE THREAD

**Issue by llaville** [megalinter/megalinter#726]
**Describe the bug**
When I analyse source code that must not raise any error with native PHPStan tool (default level is zero: see official doc at https://phpstan.org/user-guide/rule-levels), Mega-Linter show me an error

**Expected behavior**
No Error raised

**Screenshots**
```
### Processed [PHP] files
- Using [phpstan v0.12.98] https://phpstan.org/
- Mega-Linter key: [PHP_PHPSTAN]
- Rules config: [phpstan.neon]
[phpstan] src/CallbackFilterHandler.php - ERROR - 1 error(s)
--Error detail:
 ------ ----------------------------------------------------------------------- 
  Line   CallbackFilterHandler.php                                              
 ------ ----------------------------------------------------------------------- 
  81     Parameter #1 $level of method                                          
         Monolog\Handler\AbstractHandler::__construct() expects                 
         100|200|250|300|400|500|550|600|'ALERT'|'alert'|'CRITICAL'|'critical'  
         |'DEBUG'|'debug'|'EMERGENCY'|'emergency'|'error'|'ERROR'|'INFO'|'info  
         '|'NOTICE'|'notice'|'WARNING'|'warning', int|string given.             
 ------ ----------------------------------------------------------------------- 

 [ERROR] Found 1 error                                                          


❌ Linted [PHP] files with [phpstan]: Found 1 error(s) - (1.23s)

```

**Additional context**
https://github.com/nvuillam/mega-linter/blob/v4.45.0/TEMPLATES/phpstan.neon

Should fix level to zero to follow PHPStan default behavior

**Comment by llaville** [comment]
Workaround is possible:  add in your `.mega-linter.yml` config file, the following entry: `PHP_PHPSTAN_ARGUMENTS: "--level=0"`

**Comment by nvuillam** [comment]
Mega-Linter analyzes command return code, and in that case it seems that it is >0 , it looks to be phpstan default , no ?

**Comment by llaville** [comment]
No, checks at official PHPStan doc page https://phpstan.org/user-guide/rule-levels and you'll see that **The default level is 0. Once you specify a configuration file, you also have to specify the level to run.**
 (0 is the loosest and 8 is the strictest)

I think we should let final user select their analysis level. Even if [as specified](https://github.com/nvuillam/mega-linter/issues/726#issuecomment-915122066) their is a workaround
